### DISCUSSION TITLE
How insert row with uuid from JS?

### DISCUSSION THREAD

**Discussion by Kritolan** [supabase/supabase#29126] [Questions] [answered]
I have a simple table:
```sql
create table
  public.events (
    id uuid not null default gen_random_uuid(),
    title text not null
  ) tablespace pg_default;
```

I can create rows from Table Editor, but when I try to create them from JS
```js
db.from('events').insert({ title: title })
```
I get an error `invalid input syntax for type uuid: "undefined"`

For some reason, a request is sent to the address https://<SUPABASE_URL>/rest/v1/events?id=eq.undefined

**Comment by GaryAustin1** [comment, marked as answer]
Is the request you show a Post request?  It is very odd that it has a filter on it.  Looks like .eq('id', undefined) which would normally be on a select call.  What is the body of the Post request?

**Comment by Kritolan** [comment]
Well, I looked at my code more than once before creating a discussion, but only after your question I finally noticed that it had two condition branches mixed up with insert and update. Thank you, I hope this incident of inattention did not take up a lot of your time.
